### ISSUE TITLE
Desktop app title only says "Spotify Premium"

### ISSUE THREAD

**Issue by tscores** [SimonIT/spotifylyrics#23]
Unfortunately this app no longer works for me.

The app searches for the title of the Spotify window, and this now only says "Spotify Premium".

![1](https://user-images.githubusercontent.com/63917380/79641496-c98d2b80-818f-11ea-86e2-9d58af9db492.png)

This means the app is constantly searching for lyrics called "Spotify Premium". 

![2](https://user-images.githubusercontent.com/63917380/79641505-d3169380-818f-11ea-9cdb-e8f6afaace85.png)

**Comment by SimonIT** [comment]
That's odd.
Maybe the artist and the song name are not in the title because you're playing the song on another device? I think I had the same situation some time ago
_Spotify Premium_ detected as song is a real problem. The problem is that didn't experienced this problem and I own Spotify Premium, too. So until you provide me a detailed description how to reproduce it, I can nothing do.

**Comment by Arecsu** [comment]
There are certain moments that I get this behaviour. For instance, when I change from an album to another and play a song, it won't refresh the title until I get to the next song and I go back to the previous one. The same thing when I try to play one song from the search results or artist pages. Always the first one.

With first one I mean, the first song after I change the view in Spotify, not the first in the list.

What about this library? https://github.com/SwagLyrics/SwSpotify

**Comment by SimonIT** [comment]
Yes, the SwSpotify library uses the same method to get the current playing song.

Interesting idea to get the information from this panel. The problem is, I don't even know how it is properly called so I don't know how to find a way to get the information.

Another idea is to use the api spotify provides. A downside would be a lot network requests because we would have to check for song changes and the user must connect the program with spotify.

I personally would prefer to use your idea.
